site/gatsby-site/src/utils/cloudinary.js

```diff
@@ -34,7 +40,32 @@ const Image = ({ publicID, className, alt, transformation = null, plugins = [laz
 
   image.transformation = tmpImage.transformation.toString();
 
-  return <AdvancedImage alt={alt} className={className} cldImg={image} plugins={plugins} />;
+  useEffect(() => {
+    const useFallbackIfLoadFailed = () => {
+      const img = imageElement.current?.imageRef.current;
+
+      if (!(img?.naturalHeight > 0)) {
+        if (img?.complete) {
+          setCloudinaryID('fallback.jpg');
+        } else {
+          setTimeout(useFallbackIfLoadFailed, 1000);
```

Comment:
I think this needs a cleanup function:
 - component mounts
 - time out is started
 - user navigates away from the page
 - component is unmounted
 - timeout will still execute 
 - memory leak

Reply:
Added